### PR TITLE
MAP-943 weekly cancelled court moves

### PR DESCRIPTION
**Pull request by jonesj2193**
### Jira link

MAP-943

### What?

I have added/removed/altered:

- [ ]  Added config and cronjob files to automate the weekly cancelled moves report.

### Why?

I am doing this because:

- It was previously being managed on a local machine

### CODE REVIEW

File: reports/weekly-cancelled-court-moves/01-weekly-cancelled-court-moves-config.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: weekly-cancelled-court-moves-config
+data:
+  from-date: "1 day ago"
+  to-date: "7 days"
+  subject: "BaSM Cancelled Court Moves Report [TODAY_DATE]"
+  body: "BaSM Cancelled Court Moves report for yesterday ([FROM_DATE_FULL]) to [TO_DATE_FULL] is available at the link below."
+  filename: "basm-cancelled-moves-report-[TODAY_DATE]"
```
Comment (danbenton-mojdt) [inline reports/weekly-cancelled-court-moves/01-weekly-cancelled-court-moves-config.yaml:10]: Can we add court into the filename?

File: reports/weekly-cancelled-court-moves/02-weekly-cancelled-court-moves-cronjob.yaml [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  jobTemplate:
+    spec:
+      ttlSecondsAfterFinished: 345600
+      backoffLimit: 0
+      activeDeadlineSeconds: 7200
+      template:
+        spec:
+          serviceAccountName: "book-a-secure-move-api"
+          containers:
+            - name: weekly-cancelled-moves
```
Comment (danbenton-mojdt) [inline reports/weekly-cancelled-court-moves/02-weekly-cancelled-court-moves-cronjob.yaml:20]: court

**Review by danbenton-mojdt — CHANGES_REQUESTED** [review]

**Review by danbenton-mojdt — APPROVED** [review]

File: reports/weekly-cancelled-court-moves/02-weekly-cancelled-court-moves-cronjob.yaml [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+                  mountPath: /scripts
+                  readOnly: true
+                - name: report-sql
+                  mountPath: /report
+                  readOnly: false
+              env:
+                - name: DB_INSTANCE
+                  valueFrom:
+                    secretKeyRef:
+                      name: read-rds-instance-hmpps-book-secure-move-api-preprod
```
Comment (jimbali) [inline reports/weekly-cancelled-court-moves/02-weekly-cancelled-court-moves-cronjob.yaml:34]: Is preprod correct here, or should it be prod?

Comment (danbenton-mojdt) [reply reports/weekly-cancelled-court-moves/02-weekly-cancelled-court-moves-cronjob.yaml:34]: preprod is fine for the moment as we're manually applying anyway - Helm will handle namesapces when you pick that up with Josh :)

**Pull request merged**